### PR TITLE
EHV table query highlighting

### PR DESCRIPTION
**Pull request by rabstejnek**
Text matching the query is now highlighted in the EHV table:
image

### CODE REVIEW

File: frontend/animal/EhvBrowser/Table.js [hunk 1]
```diff
@@ -7,6 +7,24 @@ const MAX_ROWS = 150;
 @inject("store")
 @observer
 class Table extends Component {
+    highlight(text) {
```
Comment (shapiromatron) [inline frontend/animal/EhvBrowser/Table.js:10]: I did this recently, would it make sense to make a `highlightPatternText` method?

https://github.com/shapiromatron/hawc/blob/8f7f74a10b61350bc3227de0edb615b344ee89fd/frontend/assets/shared/components/AutocompleteTerm/index.js#L78

You could even create the regex  outside the function call, so it'd be faster.

**Review by shapiromatron — CHANGES_REQUESTED** [review]
consider using the `boldPatternText` function pattern which uses a regex instead

**Review by shapiromatron — APPROVED** [review]
@rabstejnek look ok?

**Comment by rabstejnek** [comment]
@shapiromatron looks good!

**Pull request merged**